packages/medusa/src/api-v2/admin/workflows-executions/route.ts

```diff
@@ -0,0 +1,43 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IWorkflowEngineService } from "@medusajs/workflows-sdk"
+import { MedusaRequest, MedusaResponse } from "../../../types/routing"
+
+export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
+  const workflowEngineService: IWorkflowEngineService = req.scope.resolve(
+    ModuleRegistrationName.WORKFLOW_ENGINE
+  )
+
+  const listConfig = req.listConfig
+
+  const { transaction_id, workflow_id } = req.filterableFields
+
+  const transactionIds = Array.isArray(transaction_id)
+    ? transaction_id
+    : [transaction_id]
+  const workflowIds = Array.isArray(workflow_id) ? workflow_id : [workflow_id]
```

Comment:
thought(non-blocking): Isn't this handled by the module service?

Reply:
Good call, I'll bring it to the service layer.